ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/Server.java

```diff
@@ -331,7 +331,7 @@ private static void setTlsOptions(
       } else if (keyStoreType.equals(KsqlRestConfig.SSL_STORE_TYPE_JKS)) {
```

Comment:
Could you also use equalsIgnoreCase() here? We probably need it too.

Reply:
Ready @spena

Reply:
Cool, thanks. Btw, the `KsqlConfig` has several changes, most of them are tab-spaces. Could you fix them too?